src/besselj.jl

```diff
@@ -114,34 +217,29 @@ function besselj1(x::Float64)
     x = abs(x)
     isinf(x) && return zero(x)
```

Comment:
We should probably be consistent with the structure between `besselj0` and `besselj1`. Either have the isinf check in the large argument branch (where I check if xinv is zero) or just have it at beginning.